docs/index.asciidoc

```diff
@@ -225,75 +135,46 @@ include::{plugins-repo-dir}/plugins/filters.asciidoc[]
 include::{plugins-repo-dir}/plugins/codecs.asciidoc[]
 
 // FAQ and Troubleshooting
-
 :edit_url!:
 include::static/best-practice.asciidoc[]
-
-:edit_url!: 
 include::static/config-details.asciidoc[]
-
-:edit_url!:
 include::static/troubleshoot/troubleshooting.asciidoc[]
 
-:edit_url:
-
 // Contributing to Logstash
-
+:edit_url:
 include::static/contributing-to-logstash.asciidoc[]
-
 include::static/input.asciidoc[]
 include::static/codec.asciidoc[]
 include::static/filter.asciidoc[]
 include::static/output.asciidoc[]
 
 // Logstash Community Maintainer Guide
-
-:edit_url!: 
 include::static/maintainer-guide.asciidoc[]
 
-// A space is necessary here ^^^
-
 // Plugin doc guidelines
-
-:edit_url: 
 include::static/doc-for-plugin.asciidoc[]
 
-// Submitting a Plugin
-
-:edit_url!: 
+// Submitting a Plugin 
 include::static/submitting-a-plugin.asciidoc[] 
-
-:edit_url!:
-include::static/listing-a-plugin.asciidoc[]
-
-:edit_url!: 
+include::static/listing-a-plugin.asciidoc[] 
```

Comment:
One other thing...did you mean to include this file, too?

Reply:
Yes. Thank you for checking. :-)